### PR TITLE
[Dubbo-6189] 优化部分泛化调用 #6189

### PR DESCRIPTION
**Pull request by sqtds**
https://github.com/apache/dubbo/issues/6189

### CODE REVIEW

File: dubbo-common/src/main/java/org/apache/dubbo/common/utils/PojoUtils.java [hunk 1]
```diff
@@ -591,6 +579,9 @@ private static Method getSetterMethod(Class<?> cls, String property, Class<?> va
                 }
             }
             if (method != null) {
```
Comment (lfzCarlosC) [inline dubbo-common/src/main/java/org/apache/dubbo/common/utils/PojoUtils.java:581]: `if (method != null && !method.isAccessible()) ` is more readable

File: dubbo-common/src/main/java/org/apache/dubbo/common/utils/PojoUtils.java [hunk 2]
```diff
@@ -254,28 +254,29 @@ public Object invoke(Object proxy, Method method, Object[] args) throws Throwabl
...
+        return new ArrayList<Object>(len);
     }
 
     private static Map createMap(Map src) {
         Class<? extends Map> cl = src.getClass();
+        int size = src.size();
         Map result = null;
         if (HashMap.class == cl) {
-            result = new HashMap();
+            result = new HashMap(size);
```
Comment (AlbumenJ) [inline dubbo-common/src/main/java/org/apache/dubbo/common/utils/PojoUtils.java:265]: pls use `size / 0.75 + 1` for `HashMap` to reduce resize

**Comment by AlbumenJ** [comment]
@sqtds hi, thanks for your contribution.

Please resolve confilcts with the latest master branch.

**Comment by AlbumenJ** [comment]
close for long time no response

Please feel free to reopen if you have any question

**Pull request closed without merging**